docs/pages/Configuration.md

```diff
@@ -96,25 +96,144 @@ However, it is recommended that modules include their namespace in the virtual p
 
-Your projects `*.properties` files will be read in the given encoding and any non-ASCII characters replaced with the respective unicode escape sequences. This allows you to deploy the resulting files to any environment, independent from how it expects `*.properties` files to be encoded. Please refer to [RFC 7](https://github.com/SAP/ui5-tooling/blob/master/rfcs/0007-properties-file-encoding.md) for details.
+Specification Version | Default propertiesFileSourceEncoding
+--- | ---
+**0.1, 1.0 or 1.1** | `ISO-8859-1` 
+**2.0+** | `UTF-8`
+
+If your project uses a different encoding for `*.properties` files, you need to set the `propertiesFileSourceEncoding` configuration property.
+
+The UI5 Tooling will read the corresponding files of the project in the given encoding. Any non-ASCII characters will be replaced with the respective Unicode escape sequences. This allows you to deploy the resulting files to any environment, independent from how it expects `*.properties` files to be encoded. Please refer to [RFC 7](https://github.com/SAP/ui5-tooling/blob/master/rfcs/0007-properties-file-encoding.md) for details.
 
 ````yaml
 resources:
   configuration:
-    propertiesFileSourceEncoding: UTF-8
+    propertiesFileSourceEncoding: UTF-8|ISO-8859-1
 ````
 
+## Framework Configuration
+
+!!! info
+    This configuration is available since UI5 CLI [`v2.0.0`](https://github.com/SAP/ui5-cli/releases/tag/v2.0.0)
+    and applies only to projects defining [Specification Version](#specification-versions)
+    2.0 or higher.
+
+Define your projects framework dependencies.
+
+Your projects framework configuration you must define whether you want to use the OpenUI5 or the SAPUI5 framework. Also see our [documentation on the differences between OpenUI5 and SAPUI5](./SAPUI5.md#differences-between-openui5-and-sapui5).
+
+**SAPUI5**
+```yaml
+framework:
+  name: SAPUI5
+```
+
+**OpenUI5**
+```yaml
+framework:
+  name: OpenUI5
+```
+
+If you are not sure which framework is right for you, see our [documentation on the differences between OpenUI5 and SAPUI5](./SAPUI5.md#differences-between-openui5-and-sapui5).
+
+!!! warning
+    Projects that use the OpenUI5 framework can not depend on projects that use the SAPUI5 framework.
+
+If you want to execute UI5 CLI commands directly in your project you also need to specify the framework version you want to use. Whenever you execute a UI5 CLI command, the framework version of the current root project is used.
+
+```yaml
+framework:
+  name: SAPUI5
+  version: 1.76.0
+```
+
+You can find an overview of the available versions for each framework here:
+
+- [SAPUI5 Version Overview](http://ui5.sap.com/versionoverview.html) (**Note:** The UI5 Tooling can only consume SAPUI5 starting with version 1.76.0.)
+- [OpenUI5 Version Overview](https://openui5.hana.ondemand.com/versionoverview.html)
+
+### Dependencies
+
+All libraries required by your project should be listed in the libraries section of the framework configuration.
```

Comment:
... must be listed ... (?)     
Bitte 'should' nicht verwenden, oder wenigstens nicht ohne detaillierte Erklärung. Meistens ist 'must' erforderlich.

Reply:
Done